### ISSUE TITLE
[Accessibility]: Notepad ARC toolkit issues

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#36511]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34198

### Problem description

- File and search icons should be labelled instead of hidden
- File items in notepad have insufficient color contrast. They should be made to look the same as search items.

![CleanShot 2022-06-02 at 15 21 26](https://user-images.githubusercontent.com/206864/171748163-341882f2-35c7-4be6-b24f-728e65d93523.png)

### Expected behavior

Issues are fixed as described above.

### Additional details

Note: Notepad needs to be enabled. It can be enabled by going to the "Notebooks" page and clicking on "Enable Notepad" at the top next to the title. It will then appear on the bottom right of the screen.

![CleanShot 2022-06-02 at 15 19 21](https://user-images.githubusercontent.com/206864/171747833-1f71c476-db20-4007-8e29-4ef64bdf5aa3.png)
![image](https://user-images.githubusercontent.com/206864/171747876-f6003e4b-7671-4416-af3d-cbbdb001a437.png)

Items can be added to it by running a search or viewing a file, and then opening the notepad to add the current item.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
